components/butterfly/README.md

```diff
@@ -0,0 +1,62 @@
+# Butterfly
+
+Buttefly is the Habitat gossip protocol. It's an implementation of
```

Comment:
This conversation makes me want to rename it Buttefly, just to :trollface: people